app/models/pet.rb

```diff
@@ -30,6 +30,7 @@ class Pet < ApplicationRecord
   acts_as_tenant(:organization)
 
   has_many :adopter_applications, dependent: :destroy
+  has_many :tasks, dependent: :destroy
```

Comment:
I think we can reuse "checklist_template_items" here

Reply:
@nsiwnf Thank you - I will get to work on a Task partial that more closely matches the above image. thank you for the clarification.